src/db/vco_effects.loc.tsv

```diff
@@ -21,5 +21,5 @@ effects_description_vco_ogr_greasus_ward_save	Ward save: %+n% for Greasus Goldto
 effects_description_vco_ksl_ort_luminark_unlock_dummy	Unlock new unit: Luminark of Hysh	true
 effects_description_vco_skv_food	Food generated: %n	true
 effects_description_vco_skv_per_turn_growth_vat	Growth Juice: %+n per turn	true
-effects_description_vco_skv_per_turn_mutagen	Mutagen income per turn: %+n	true
+effects_description_vco_skv_per_turn_mutagen	Mutagen %+n per turn	true
```

Comment:
Usually you want to use `:` to keep it consistent with vanilla effects. `Mutagen: %+n per turn`. Do not worry about fixing it tho, I will do myself.

Reply:
Yeh I was actually trying to make it match line 23 with the Growth Juice but I forgot the colon.